### ISSUE TITLE
[Feature]: Basic options when running shell

### ISSUE THREAD

**Issue by nithinmuthukumar** [MrPicklePinosaur/shrs#328]
## What would you like to see added?
There should be a --version and --help in shrs_example. Would be nice if there was a way to build this out of the box using the shrs library for any shell implementation.

This came up when I was trying to use neofetch in the startup hook. Neofetch calls --version on the SHELL env var to get the version, which currently just runs the shell.

other options should also be considered

**Comment by MrPicklePinosaur** [comment]
This is technically the end user's responsibility. A lot of great libraries for building clis already exist. We could definitely add command line arguments to shrs_example though, I just dont think shrs should provide anything for this out of the box.

**Comment by nithinmuthukumar** [comment]
Yeah that makes sense, will change the name of the feature request

**Comment by MrPicklePinosaur** [comment]
I suggest using the `clap` crate to do this. Most of our builtins use `clap` for arg parsing too
